packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -10,14 +17,44 @@ export const defaultMuiThemeOptions: ThemeOptions = {
   palette: palette,
   components: {
     MuiAlert: {
+      defaultProps: {
+        square: true,
+        iconMapping: {
+          error: <HighlightOffTwoTone />,
+          info: <InfoTwoTone />,
+          success: <CheckCircleTwoTone />,
+          warning: <ErrorTwoTone />,
+        },
+      },
       styleOverrides: {
         root: ({ ownerState, theme }) => ({
-          borderRadius: '0px',
-          fontSize: '16px',
+          alignItems: 'center',
+          fontSize: '14px',
           borderLeft: `10px solid ${
             // The default severity is 'success'.
             theme.palette[ownerState.severity || 'success']['main']
           }`,
+          color: theme.palette.grey[900],
+          padding: theme.spacing(2),
+          marginBottom: theme.spacing(1),
+          marginTop: theme.spacing(1),
+        }),
+        icon: ({ theme }) => ({
+          marginRight: theme.spacing(3),
+        }),
+        action: ({ ownerState, theme }) => ({
+          paddingRight: theme.spacing(2),
+          ...(ownerState.onClose &&
+            !ownerState.action && { color: theme.palette.grey[700] }),
```

Comment:
Style the close button, which is only included if onClose is defined, but actions are not defined, see [here](https://mui.com/material-ui/react-alert/#actions)